WinImageBuilder.psm1

```diff
@@ -582,15 +574,6 @@ function Create-VirtualSwitch
     return $sw
 }
 
-function Check-Prerequisites
-{
-    $needsHyperV = Get-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V
-    if ($needsHyperV.State -ne "Enabled"){
-        Write-Warning $noHypervWarning
-        exit 1
-    }
-}
-
 function GetOrCreateSwitch
 {
     $vmSwitches = Get-VMSwitch -SwitchType external

```

Comment:
This commit tries to solve a check fail on the host server versions, as the Get-WindowsOptionalFeature is not present on Server versions. To check if the host is a server or a client version, we ll use (Get-WmiObject win32_operatingsystem).producttype.